easybuild/easyblocks/p/perl.py

```diff
@@ -42,18 +43,27 @@
 class EB_Perl(ConfigureMake):
     """Support for building and installing Perl."""
 
+    @staticmethod
+    def extra_options():
+        """Add extra config options specific to Perl."""
+        extra_vars = {
+            'use_perl_threads': [True, "Use internal Perl threads by means of the -Dusethreads compiler directive", CUSTOM],
+        }
+        return ConfigureMake.extra_options(extra_vars)
+
     def configure_step(self):
         """
         Configure Perl build: run ./Configure instead of ./configure with some different options
         """
-        configopts = ' '.join([
-            self.cfg['configopts'],
-            "-Dusethreads",
-            '-Dcc="%s"' % os.getenv('CC'),
-            '-Dccflags="%s"' % os.getenv('CFLAGS'),
-            '-Dinc_version_list=none',
-        ])
-        cmd = './Configure -de %s -Dprefix="%s" ' % (configopts, self.installdir)
+        configopts = []
+        configopts.append(self.cfg['configopts'])
+        configopts.append('-Dcc="{0}"'.format(os.getenv('CC')))
+        configopts.append('-Dccflags="{0}"'.format(os.getenv('CFLAGS')))
+        configopts.append('-Dinc_version_list=none')
```

Comment:
I didn't mean to imply you should use `.append` all over the place... just define the initial `configopts` in one go, and append `-Dusethreads` if desired

see https://github.com/valtandor/easybuild-easyblocks/pull/1